### PR TITLE
test(date-range-picker-component): add test coverage

### PR DESCRIPTION
**Pull request by vdtdev**
AOS2-29: Add test coverage for DateRangePickerComponent (https://bitovi.atlassian.net/browse/AOS2-29?atlOrigin=eyJpIjoiZDU0ZGRjOTVjMGEwNGE2ODg0NDI0ZTk3NWI2NzRkMjAiLCJwIjoiaiJ9)

### CODE REVIEW

**Review by krivanek06 — APPROVED** [review]
LGTM

File: src/app/components/date-range-picker/date-range-picker.component.spec.ts [hunk 1]
```diff
@@ -74,7 +76,176 @@ describe('DateRangePickerComponent', () => {
...
 	it('should create', () => {
 		expect(component).toBeTruthy();
 	});
+
+	it('should have a minDate of 1/1/2015', () => {
+		expect(component.minDate).toStrictEqual(new Date(2015, 0, 1));
+	});
+
+	describe('when dateRangeFormGroup value changes', () => {
+		// beforeEach(() => {
```
Comment (m-thompson-code) [inline src/app/components/date-range-picker/date-range-picker.component.spec.ts:94]: Can we remove the commented out code?

File: src/app/components/date-range-picker/date-range-picker.component.spec.ts [hunk 2]
```diff
@@ -74,7 +76,176 @@ describe('DateRangePickerComponent', () => {
...
+			const dateService = TestBed.inject(DateService);
+			jest.spyOn(dateService, 'isValidDateRange').mockReturnValue(true);
+			jest.spyOn(component.dateRangeFormGroup, 'setValue').mockImplementation(() => ({}));
+
+			const dateRange: DateRange = [new Date('2021-10-01'), new Date('2021-11-01')];
+
+			component.writeValue(dateRange);
+
+			// Wait some time for the timeout
+			tick(500);
```
Comment (m-thompson-code) [inline src/app/components/date-range-picker/date-range-picker.component.spec.ts:215]: nice

**Review by m-thompson-code — CHANGES_REQUESTED** [review]
very minor code change request: remove commented out code :P

**Review by m-thompson-code — APPROVED** [review]
woop woop~

**Pull request merged**